docs/evaluation/scientific-knowledge.md

```diff
@@ -16,18 +16,100 @@ More details are coming soon!
+                    "++chat_template_kwargs.reasoning_effort=high ++code_execution=true "
+                    "++parse_reasoning=True "
+                    '\'++end_reasoning_string="<|start|>assistant<|channel|>final<|message|>"\''
+        ),
+        cluster=cluster,
+        expname="simpleqa-gpt-oss-120b-tool-output-only",
+        model="/hf_models/gpt-oss-120b",
+        server_type="vllm",
+        server_gpus=8,
+        server_args="--async-scheduling",
+        benchmarks="simpleqa:2",
+        split="verified",
+        output_dir="/workspace/simpleqa-gpt-oss-120b-tool-output-only",
+        with_sandbox=True,
+        judge_model="/hf_models/gpt-oss-120b",
+        judge_server_type="vllm",
+        judge_server_gpus=8,
+        judge_server_args="--async-scheduling  --reasoning-parser GptOss",
+    )
+    ```
+
+
+
+#### Configuration: `gpt-oss-120b` without tool
+
+=== "python interface"
+
+    ```python
+    from nemo_skills.pipeline.cli import wrap_arguments, eval
+    cluster = 'slurm'
+    eval(
+        ctx=wrap_arguments(
+                    "++inference.temperature=1.0 ++inference.tokens_to_generate=65536 "
+                    "++inference.extra_body.reasoning_effort=high "
+        ),
+        cluster="ord",
+        expname="simpleqa-gpt-oss-120b-notool",
+        model="/hf_models/gpt-oss-120b",
+        server_type="vllm",
+        server_gpus=8,
+        server_args="--async-scheduling --reasoning-parser GptOss",
+        benchmarks="simpleqa:2",
+        split="verified",
+        output_dir="/workspace/simpleqa-gpt-oss-120b-notool",
+        judge_model="/hf_models/gpt-oss-120b",
+        judge_server_type="vllm",
+        judge_server_gpus=8,
+        judge_server_args="--async-scheduling  --reasoning-parser GptOss",
+    )
+    ```
+Note: The module name for `reasoning-parser` differs across `vllm` versions. Depending on your version, it might appear as `openai_gptoss` or `GptOss`. In the latest main branch, it is named `openai_gptoss`. You can verify this in [gptoss_reasoning_parser.py](https://github.com/vllm-project/vllm/blob/main/vllm/reasoning/gptoss_reasoning_parser.py) and confirm which version your environment uses.
+
+#### Result
+
+We also tested a variant where the full generation output was provided to the judge—disabling both "parse_reasoning" and "end_reasoning_string". This configuration, labeled `simpleqa-gpt-oss-120b-tool-full-generation`, produced results nearly identical to the standard setup where the reasoning portion is excluded from the judge’s input.
+
+
+
+| Run Name                                      |     pass@1 |   majority@2 |    pass@2 |
+|:----------------------------------------------|-----------:|-------------:|----------:|
+| simpleqa-gpt-oss-120b-notool                  | 12.93     |   12.93     | 17.22   |
+| simpleqa-gpt-oss-120b-tool-full-generation                    | 80.30    |   80.30    | 84.78   |
```

Comment:
please make sure to render docs (with mkdocs serve) and double check that formatting looks good